modules/plugins/filetree/nvimtree/config.nix

```
@@ -77,22 +77,29 @@ in {
               -- buffer is a real file on the disk
               local real_file = vim.fn.filereadable(data.file) == 1
 
+              -- buffer is a directory 
+                local directory = vim.fn.isdirectory(data.file) == 1
+
               -- buffer is a [No Name]
                 local no_name = data.file == "" and vim.bo[data.buf].buftype == ""
 
               -- &ft
               local filetype = vim.bo[data.buf].ft
 
               -- only files please
-              if not real_file and not no_name then
+              if not real_file and not directory and not no_name then
                 return
               end
 
               -- skip ignored filetypes
               if vim.tbl_contains(IGNORED_FT, filetype) then
                 return
-              end
+              end 
```

```suggestion
              end
```

no space at end of line pls